server/controllers/users.py

```diff
@@ -32,11 +33,13 @@ def mentor_rankings():
         unrated_tickets = Ticket.query.filter_by(claimant=user, status=3).all()
         totalUnrated += len(unrated_tickets)
 
-        if (totalRatings > 0):
-            ret.append({"name": user.name, "rating": "{:.1f}".format(
+        if (totalRatings > 0): # "{:.1f}".format
```

Comment:
are u not doing precision anymore? Would be careful about that.

Reply:
was moved to frontend in latest commit, moved back here in revisions. had to keep as float since we were sorting